src/components/Codec.css

```diff
@@ -2,28 +2,81 @@
     box-sizing: border-box;
 }
 
-.invalid {
-    background-color: #e84545;
+.title {
+    font-weight: bold;
+    font-size: 120%;
+    padding: 4px 0;
```

Comment:
this applies to all here, but any sizing should be in `rem` with the assumption that you are styling for a 1080p display. All values you use should be cleanly divisible by 3, (`v % 3 == 0`). This is so that a 1080p layout can be scaled down to 720p and retain integer values, rather than floats with a high mantissa.

I'll add a note to the readme about this.

Reply:
REM is understandable from old device as well?

Reply:
@JohnPaulHarold can you please explain to me why we should use the sizing for border and padding in rem? I get that every sizing should be a multiple of 3. But why can't we just use pixels as measure of unit? 3px? 6px? If we need to rely on rem then we are forced to use 24px font size for 1080p and 12px for 720p otherwise there's no way to obtain multiples of 3 using rem

Reply:
@federicoportoghese because 3px at 1080p is 3px at 720p. Using rems, means that a widget with, say, 480px wide @ 1080p equates to 320px wide @ 720p. Using rems as units means that the layout remains proportionally correct.

You _could_ use pixels as units, but then you'd need a lot of media queries to deal with all the changes. Or you could get your server to send you the right CSS files for your viewport based on your user agent. I'd rather not do the former solution, and I've not written anything for the other solution.

Previously, I've used a `pxToRem()` type sass function, so you don't need to do all this manual calculation, and you can just enter the pixels values and let the compiler create all the rems for you. I'm not averse to using sass, or such a function, but I just haven't got around to it yet.